test/cql-pytest/test_json.py

```diff
@@ -159,18 +159,12 @@ def test_fromjson_varint_empty_string_prepared(cql, table1):
     with pytest.raises(FunctionFailure):
         cql.execute(stmt, [p, '""'])
 
-# Cassandra allows the strings "true" and "false", not just the JSON constants
-# true and false, to be assigned to a boolean column. However, very strangely,
-# it only allows this for prepared statements, and *not* for unprepared
-# statements - which result in an InvalidRequest!
-# Reproduces #7915.
 def test_fromjson_boolean_string_unprepared(cql, table1):
     p = unique_key_int()
-    with pytest.raises(InvalidRequest):
-        cql.execute(f"INSERT INTO {table1} (p, b) VALUES ({p}, '\"true\"')")
-    with pytest.raises(InvalidRequest):
-        cql.execute(f"INSERT INTO {table1} (p, b) VALUES ({p}, '\"false\"')")
-@pytest.mark.xfail(reason="issue #7915")
```

Comment:
Here you are right to remove the xfail line, but please add a "`# Reproduces #7915`" comment instead. I think it's useful when you look at tests in the future to understand why they were written, and what they are trying to reproduce.